protocol/v1/qbft/types.go

```diff
@@ -38,33 +38,36 @@ type State struct {
 	// lambda is an instance unique identifier, much like a block hash in a blockchain
 	Identifier atomic.Value // []byte
 	// Height is an incremental number for each instance, much like a block number would be in a blockchain
-	Height        atomic.Value // specqbft.Height
-	InputValue    atomic.Value // []byte
-	Round         atomic.Value // specqbft.Round
-	PreparedRound atomic.Value // specqbft.Round
-	PreparedValue atomic.Value // []byte
+	Height                          atomic.Value // specqbft.Height
+	InputValue                      atomic.Value // []byte
+	Round                           atomic.Value // specqbft.Round
+	PreparedRound                   atomic.Value // specqbft.Round
+	PreparedValue                   atomic.Value // []byte
+	ProposalAcceptedForCurrentRound atomic.Value // *specqbft.SignedMessage
```

Comment:
@nkryuchkov this is just to verify propose on commit?

Reply:
@nivBlox it's also used [on prepare](https://github.com/bloxapp/ssv-spec/blob/main/qbft/prepare.go#L14) and [on proposal](https://github.com/bloxapp/ssv-spec/blob/main/qbft/proposal.go#L93).